src/EnergyPlus/OutputProcessor.cc

```diff
@@ -187,7 +187,7 @@ namespace OutputProcessor {
         state.files.mtd.ensure_open(state, "InitializeMeters", state.files.outputControl.mtd);
     } // InitializeOutput()
 
-    void addEndUseSubcategory(EnergyPlusData &state, SOVEndUseCat sovEndUseCat, std::string_view const endUseSubName)
+    void addEndUseSubcategory(EnergyPlusData &state, EndUseCat endUseCat, std::string_view const endUseSubName)
```

Comment:
Removed `SOV` prefixes after indirections were eliminated. No need to differentiate between `SOV` version of an enum and internal version of same enum.